packages/core/cli/src/__template__/src/components/Template.cy.ts

```diff
@@ -1,7 +1,6 @@
 // Cypress component test spec file
 
 import {%%COMPONENT_NAME%%} from './{%%COMPONENT_NAME%%}.vue'
-import { mount } from 'cypress/vue'
```

Comment:
is this legit removal?

Reply:
Yes, in the component tests, we use a custom `cy.mount` so we don't need to import the default one. I noticed it when I created the new package.

Reply:
I think so tho I'll let @adamdehaven respond to this one